packages/ui/src/theme/overwrites/index.ts

```diff
@@ -1,11 +1,13 @@
 import { extendTheme } from '@chakra-ui/react';
 
+import { components } from './components';
 import { foundations } from './foundations';
 import { styles } from './styles';
 
 export const theme = extendTheme({
   ...foundations,
   styles,
+  components: { ...components },
```

Comment:
There is no need to destructure `components` to a new object. You can just pass them as `styles` are passed